manifests/metadata.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
The one in the root directory is needed by the forge, the one in the manifests directory is incorrect (and out of date looking at the version)